### ISSUE TITLE
Can't make it work

### ISSUE THREAD

**Issue by romu70** [retspen/webvirtcloud#15]
Hi,
I've installed WebVirtCloud to make some testing. Setup ran fine as much as I can say. If I browse http://localhost, I see the nginx welcome page. But I've tried ports 8000 (like webvirtmgr) and 6080 (taken from the settings.py), none work.

Is webvirtcloud autonomous? Or Do I have to install it with webvirtmgr?

**Comment by retspen** [comment]
Show your nginx settings for webvirtcloud virtual host

**Comment by romu70** [comment]
Which file ? FYI, in the /etc/nginx/sites-enabled/default, there is no virtual host at all, just the default nginx configuration on port 80.

**Comment by retspen** [comment]
https://github.com/retspen/webvirtcloud#install-webvirtcloud-panel

**Comment by romu70** [comment]
Yes, this is what I ran yesterday when I installed it. I'll era my current setup and re-run from scratch. Let you know.

**Comment by retspen** [comment]
In file webvirtmgr.conf for nginx you have parameter "server_name
webvirtcloud.example.com;” you need change it

**Comment by romu70** [comment]
So, I re-ran the setup. The /etc/nginx/conf.d/webvirtcloud.conf file is:

```
server {
    listen 80;

    server_name webvirtcloud.example.com;
    #access_log /var/log/nginx/webvirtcloud-access_log; 

    location /static/ {
        root /var/www/webvirtcloud;
        expires max;
    }

    location / {
        proxy_pass http://127.0.0.1:8000;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-for $proxy_add_x_forwarded_for;
        proxy_set_header Host $host:$server_port;
        proxy_set_header X-Forwarded-Proto $remote_addr;
        proxy_connect_timeout 600;
        proxy_read_timeout 600;
        proxy_send_timeout 600;
        client_max_body_size 1024M;
    }
}
```

And still doesn't work.

**Comment by retspen** [comment]
Change parameter   "server_name webvirtcloud.example.com;" - this is virtual host name

**Comment by romu70** [comment]
Changed to my machine name and restarted both nginx and supervisord. Browsing http://<server_name value> leads to the nginx default page. http://<server_name value>:8000 leads to "Unable to connect".

**Comment by retspen** [comment]
http://your_server_name/ - without 8000

**Comment by romu70** [comment]
Just the "Welcome to nginx" page.

**Comment by retspen** [comment]
What show?

```bash
$ ping your_server_name
```

**Comment by romu70** [comment]
127.0.1.1 because I'm connected localy to this server.

**Comment by retspen** [comment]
Ok, change <code>server_name xxxx;</code> to <code>server_name localhost;</code> and restart nginx service. If you would have problem you need check default config for nginx where define sever_name localhost.

**Comment by romu70** [comment]
Does not change anything, still see the nginx default page.

**Comment by retspen** [comment]
Do you connect to http://localhost ?

**Comment by romu70** [comment]
Yes of course. I'll test with webvirtmgr to see if it's better.

**Comment by retspen** [comment]
Can you setup nginx like in webvirtmgr, settings for nginx the same

**Comment by romu70** [comment]
Ok, removed all files within /var/www, and ran again the installation process, and it works! Don't know where the problem was. Just an issue, it seems to not process the CSS, there is no style on the page.

**Comment by retspen** [comment]
Check ```/static/``` path in config must be ```root /var/www/webvirtcloud;```

**Comment by romu70** [comment]
Path seems right in the HTML, but the Firefox inspector says: "No CSS Prpperties found".

I've also tried to change "webvirtcloud.example.com" to my machine name in the /etc/nginx/conf.d/webvirtcloud.conf file, but then, I get the "Unable to connect" page again. Only localhost gives me something, but without CSS.

**Comment by retspen** [comment]
Show your nginx error log

**Comment by NerdsvilleCEO** [comment]
This has been fixed in the latest merges :)

**Comment by aa1121** [comment]
I'm having a similar issue. I can only connect through localhost using port forwarding, connecting to the remote ip address gives me nothing, and css doesnt load. How can i fix this?, either get the css to load or connect directly through the ip address

**Comment by catborise** [comment]
hi @aa1121 could you please create a vm on the host. then install webvirtcloud on it.

**Comment by aa1121** [comment]
i already have vms on the host, reinstalled webvirtcloud but i'm getting the same issue

**Comment by catborise** [comment]
@aa1121 install webvirtcloud on a vm not to host

**Comment by aa1121** [comment]
did that, same thing, no css...

**Comment by catborise** [comment]
whic os do you use for webvirtcloud installation?

**Comment by aa1121** [comment]
in host: debian, in vm ubuntu

**Comment by catborise** [comment]
i try every installation method like vagrant, docker and normal. all of them works as expected.  what causes you problem, i do not know.  may be you can try these methods: https://github.com/retspen/webvirtcloud/wiki

**Comment by aa1121** [comment]
I can now access the web interface by connecting to the remote ip address, but if i try to connect through localhost using port forwarding, it still gives me issue where css doesnt load... I found a workaround for that though, i run this command on the remote machine "python /srv/webvirtcloud/manage.py runserver 0:8001 --settings=webvirtcloud.settings-dev" and connect to localhost:8001 on the local machine, with port forwarding enabled. This way, css loads

**Comment by aa1121** [comment]
well never mind, this doesnt work for me anymore, i'll try to get it working again and then i'll update with instructions

**Comment by aa1121** [comment]
ok, these are the instructions that worked for me in order to get it working through localhost using port forwarding, with css loading

cd /srv/webvirtcloud
source venv/bin/activate
python /srv/webvirtcloud/manage.py runserver 0:8002 --settings=webvirtcloud.settings-dev

you might need root access or not, depending on permissions
